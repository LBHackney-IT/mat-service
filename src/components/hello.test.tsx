import * as React from 'react';
import { shallow } from 'enzyme';
import Hello from './hello';

describe('Hello', () => {
	it('Renders Nigel', () => {
		const hello = shallow(<Hello name="Nigel" greeting="Hello"></Hello>);
		expect(hello.html()).toBe("<div class=\"hello\">Hello Nigel, you have 0 tenancies!</div>");
		expect(hello.find('.hello').length).toBe(1);
		expect(hello).toMatchSnapshot();
	});

	it('Renders Matt', () => {
		const hello = shallow(<Hello name="Matt" greeting="Hello"></Hello>);
		expect(hello.html()).toBe("<div class=\"hello\">Hello Matt, you have 3 tenancies!</div>");
		expect(hello.find('.hello').length).toBe(1);
		expect(hello).toMatchSnapshot();
	});
});
