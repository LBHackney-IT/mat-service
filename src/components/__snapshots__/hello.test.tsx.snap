// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello Renders Matt 1`] = `ShallowWrapper {}`;

exports[`Hello Renders Nigel 1`] = `ShallowWrapper {}`;
